build_tools/build/src/engine/context.rs

```diff
@@ -574,6 +580,59 @@ impl RunContext {
 
         Ok(())
     }
+
+    async fn stdlib_api_check(&self, built_enso: &BuiltEnso) -> Result {
+        for libname in ["Base"] {
```

Comment:
- One way is to hardcode list of libraries here
- other option would be to list all directories in `ls -d ./distribution/lib/Standard/*/*/docs/api` and use those...
- the benefit of the later approach is: nobody would need to change Rust code to enable testing of some other library
- it would be enough to repeat #12203 for another library

Reply:
Implemented your suggestion in 053191876711f21c8f4efc0da1b2ac5953d634e4. The list is not hardcoded now. This is a portion of output of the command now:
```
...
DEBUG main_internal: enso_build::engine::context: Running standard libraries API check.
DEBUG main_internal: enso_build::engine::context: Checking API for standard libraries: [StdLib { name: "Base", path: "/home/pavel/dev/enso/distribution/lib/Standard/Base/0.0.0-dev" }]
DEBUG main_internal: enso_build::engine::context: Checking API for library Standard.Base, its API dir is in "/home/pavel/dev/enso/distribution/lib/Standard/Base/0.0.0-dev/docs/api"
...
```

Obviously, only `Standard.Base` is considered for the API check.